src/com/farzaneh/servlet/UserServlet.java

```diff
@@ -0,0 +1,78 @@
+package com.farzaneh.servlet;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.UUID;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.farzaneh.exception.ProcessException;
+import com.farzaneh.model.Person;
+import com.farzaneh.model.Product;
+import com.farzaneh.model.ProductCategory;
+import com.farzaneh.service.dao.PersonDAO;
+import com.farzaneh.service.dao.ProductDAO;
+
+@WebServlet("/UserServlet")
+public class UserServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+
+	public UserServlet() {
+		super();
+	}
+
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+		doPost(request, response);
+	}
+
+	protected void doPost(HttpServletRequest request, HttpServletResponse response)
```

Comment:
Your method is too complex, try to break it into other methods or other objects

Reply:
I break this method to some separated methods